### PR TITLE
Added more error messages

### PR DESCRIPTION
**Pull request by rodolfodosanjos**

### CODE REVIEW

File: errors [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+{
+  data: {
+    maxLength: 8,
+    minLength: 7
+  },
+  errorCode: "DBA_7",
+  field: "account",
+  variable: ['maxLength', 'minLength'],
+  message: "Conta é obrigatória e deve conter entre {{minLength}} e {{maxLength}} números"
```
Comment (valtlfelipe) [inline errors]:
```suggestion
  message: "Conta é obrigatória e deve conter entre 7 e 8 números"
```

File: errors [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Code - The code starts with the initials of the error context, example: DBA_1 which belongs to DBA context.
+- Message - The message of the error.
+- Field - Additional info.
+- Variable - Additional info.
+- Data - Additional info.
+
+#### Example of response
+```
+{
+  data: {
```
Comment (rodolfodosanjos) [inline errors]:
```suggestion
```

File: errors [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    maxLength: 8,
```
Comment (rodolfodosanjos) [inline errors]:
```suggestion
```

File: errors [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    minLength: 7
```
Comment (rodolfodosanjos) [inline errors]:
```suggestion
```

File: errors [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  },
```
Comment (rodolfodosanjos) [inline errors]:
```suggestion
```

File: errors [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+- Field - Additional info.
+- Variable - Additional info.
+- Data - Additional info.
+
+#### Example of response
+```
+{
+  errorCode: "DBA_7",
+  field: "account",
+  variable: ['maxLength', 'minLength'],
```
Comment (rodolfodosanjos) [inline errors]:
```suggestion
```

File: errors [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  message: "Conta é obrigatória e deve conter entre 7 e 8 números"
+}
+```
+
+## Errors
+**Some errors have more than one message, these multiple error messages will be removed until august 12th leaving all errors with only one message**
+| Code     | message                                                                                         |
+|----------|-------------------------------------------------------------------------------------------------|
+| DBA_1    | "Dígito da agência é obrigatório e deve ser numérico ou x"                                      |
+| DBA_2    | "Dígito da conta é obrigatório e deve ser numérico ou x"                                        |
```
Comment (t-bonatti) [inline errors]: spaces?

Comment (negherbon) [reply errors]: you are crazy man?

File: errors [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+```
+{
+  errorCode: "DBA_7",
+  field: "account",
+  message: "Conta é obrigatória e deve conter entre 7 e 8 números"
+}
+```
+
+## Errors
+**Some errors have more than one message, these multiple error messages will be removed until august 12th leaving all errors with only one message**
```
Comment (negherbon) [inline errors]:
```suggestion
**Some errors code have more than one message, these multiple error messages will be removed until august 12th leaving all errors with only one message**
```

**Pull request merged**